next.config.mjs

```diff
@@ -156,45 +156,43 @@ const nextConfig = {
   },
 };
 
-export default withSentryConfig(
-  nextConfig,
-  {
-    // For all available options, see:
-    // https://github.com/getsentry/sentry-webpack-plugin#options
-    // Cannot use `isProduction` here, as this file is not supporting ts
-    enabled: process.env.ENV_NAME === 'prod',
-    // Suppresses source map uploading logs during build
-    silent: true,
-    org: 'jumper-exchange',
-    project: 'jumper-front',
-  },
-  {
-    // For all available options, see:
-    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
+export default withSentryConfig(nextConfig, {
```

Comment:
According to the `withSentryConfig` type definition it should take a single config object